nuxt.config.ts

```diff
@@ -61,4 +61,6 @@ export default defineNuxtConfig({
       rApiBase: "", // https://nuxt.com/docs/getting-started/testing#registerendpoint
     },
   },
+
+  compatibilityDate: "2024-12-11",
```

Comment:
I'm mainly adding this in to silence the warning about it that happens on starting up the dev server, on the theory that not doing so causes it to be added dynamically, which possibly triggers a full app re-load on start-up (as updating this config normally does), which would slow down the development experience.

Docs: https://nuxt.com/docs/api/nuxt-config#compatibilitydate